public/style.css

```diff
@@ -0,0 +1,106 @@
+.app_name {
+  font-family: "Motiva Sans", Sans-serif;
+  font-weight: normal;
+  padding-left: 0px;
+  color: white;
+  font-size: 26px;
+  line-height: 32px;
+  text-overflow: ellipsis;
+  margin-left: 157px;
+  margin-top: 30px;
+}
+
+.background {
+  background-color: #1b2838;
+}
+
+br {
+  content: "";
+  margin: 3em;
+  display: block;
+  font-size: 24%;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+}
+
+.description {
+  display: inline-block;
+  font-family: Arial, Helvetica, sans-serif;
+  color: #c6d4df;
+  max-height: 111px;
+  font-size: 13px;
+  line-height: 18px;
+}
+
+div {
+  display: block;
+}
```

Comment:
Setting all `<div>` elements across the board as `block` seems a bit drastic, targeting base elements directly is generally something to avoid. You might want to use a specific class for these elements instead or targeting the `<div>` elements specifically within your widget.